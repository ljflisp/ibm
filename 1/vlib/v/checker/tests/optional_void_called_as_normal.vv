fn foo() ? {
	println('foo is called')
}

fn main() {
	// Calling foo() without ? or an or block, should be an error.
	foo()
}
