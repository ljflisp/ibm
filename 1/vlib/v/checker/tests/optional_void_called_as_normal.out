vlib/v/checker/tests/optional_void_called_as_normal.vv:7:2: error: foo() returns an option, so it should have either an `or {}` block, or `?` at the end
    5 | fn main() {
    6 |     // Calling foo() without ? or an or block, should be an error.
    7 |     foo()
      |     ~~~~~
    8 | }
